fictadvisor-api/src/v2/api/controllers/ResourceController.ts

```diff
@@ -150,61 +83,22 @@ export class ResourceController {
     return this.resourceMapper.getResource(updResource);
   }
 
-  @ApiCookieAuth()
-  @ApiOkResponse({
-    type: ResourcesResponse,
-  })
-  @ApiBadRequestResponse({
-    description: `\n
-    InvalidBodyException:
-      Name is too short (min: 3)
-      Name is too long (max: 50)
-      Resource id can not be empty
-      
-    InvalidEntityId:
-      Resource with such id is not found`,
-  })
-  @ApiForbiddenResponse({
-    description: `\n
-    NoPermissionException:
-      You do not have permission to perform this action`,
-  })
-  @ApiUnauthorizedResponse({
-    description: `\n
-    UnauthorizedException:
-      Unauthorized`,
-  })
   @ApiEndpoint({
     summary: 'Update many student resources by their id`s',
+    documentation: ResourceDocumentation.UPDATE_MANY,
     permissions: PERMISSION.RESOURCES_UPDATE,
   })
   @Patch()
   async updateMany (
-      @Body() body: UpdateResourcesDTO,
+    @Body(ValidateResourcesPipe) body: UpdateResourcesDTO,
   ) {
     const updResources = await this.resourceService.updateMany(body);
     return this.resourceMapper.getResources(updResources);
   }
 
-  @ApiCookieAuth()
-  @ApiBadRequestResponse({
-    description: `\n
-    InvalidEntityId:
-      Resource with such id is not found`,
-  })
-  @ApiForbiddenResponse({
-    description: `\n
-    NoPermissionException:
-      You do not have permission to perform this action`,
-  })
-  @ApiParam({
-    name: 'resourceId',
-    description: 'Id of student resource',
-    type: String,
-    required: true,
-  })
   @ApiEndpoint({
     summary: 'Delete specific student resource',
+    documentation: ResourceDocumentation.DELETE,
     permissions: PERMISSION.RESOURCES_DELETE,
   })
   @Delete('/:resourceId')
```

Comment:
I can't understand what I need to return in delete method. It can be just void, or I need return ResourceResponse? Why I should use mapper here?
```
  @Delete('/:resourceId')
  async delete (
    @Param('resourceId', ResourceByIdPipe) resourceId: string,
  ): Promise<ResourceResponse> {
    const resource = await this.resourceService.delete(resourceId);
    return this.resourceMapper.getResource(resource);
  }
```
or just like
``` 
  @Delete('/:resourceId')
  delete (
    @Param('resourceId', ResourceByIdPipe) resourceId: string,
  ) {
    return this.resourceService.delete(resourceId);
  }
```

Reply:
`resourceService.delete` is async method, so return type will be `Promise<void>`